multimodal/medical_document_parser/app.py

```
@@ -0,0 +1,110 @@
+"""
+
+
+def _format_flagged_text(flagged_items: list[str]) -> str:
+    if not flagged_items:
+        return "No abnormal or critical values flagged."
+    return "\n".join(f"• {item}" for item in flagged_items)
+
+
+def parse_document(
+    upload: str | None,
+    progress: gr.Progress = gr.Progress(),
```

Ruff (0.15.12)

[warning] Do not perform function call `gr.Progress` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)